xcrypto/rsa_test.go

```diff
@@ -58,7 +58,7 @@ func TestRSAEncryptDecryptPKCS1(t *testing.T) {
 func TestRSAEncryptDecryptOAEP(t *testing.T) {
```

Comment:
Test that upstream `crypto/rsa` can decrypt ciphertexts (with labels) generated by us and vice-versa.